README.md

```diff
@@ -61,22 +61,22 @@ end
 
 1. Tell ActiveRecord to ignore the column from its cache
 
-  ```ruby
```

Comment:
Please keep the changeset to a minimum.

Reply:
I think that was due to my editor settings. I've reverted that portion of the change.